front/src/pages/MainPage/components/Grid/index.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import { useDispatch, useSelector } from "react-redux";
+
+import { selectBooks, addItemToBasket } from "features/books/booksSlice";
+import { checkSingleStatus } from "utils/checkSingleStatus";
+
+import { Card } from "./components";
+import { useStyles } from "./styles";
+
+export default function SpacingGrid() {
+  const dispatch = useDispatch();
+  const books = useSelector(selectBooks);
+  const classes = useStyles();
+
+  const data = checkSingleStatus(books.dataStatus);
```

Comment:
data powinno dotyczyć danych, nie statusu. I zrób to selektorem, np.

const status = useSelector(selectBooksFetchStatus);

sam selektor w komentarzu na dole opiszę

Reply:
Done.